src/SourceBuild/content/repo-projects/sdk.proj

```diff
@@ -7,6 +7,10 @@
     <BuildCommandArgs>$(BuildCommandArgs) /p:PackageProjectUrl=https://github.com/dotnet/sdk</BuildCommandArgs>
     <BuildCommandArgs>$(BuildCommandArgs) /p:PublishCompressedFilesPathPrefix=$(SourceBuiltToolsetDir)</BuildCommandArgs>
 
+    <!-- Propagate RID set in source-build to sdk repo -->
+    <BuildCommandArgs>$(BuildCommandArgs) /p:PortableRid=$(PortableRid)</BuildCommandArgs>
```

Comment:
`PortableRid` won't detect musl.

For updating the rid graph in the `runtime` repo, we've been using `NETCoreSdkPortableRuntimeIdentifier`:

https://github.com/dotnet/installer/blob/23922bc0c40ef5bf8a1705fe810e0f7a359d3ffb/src/SourceBuild/content/repo-projects/runtime.proj#L21-L22